stix_shifter_modules/crowdstrike/stix_transmission/api_client.py

```diff
@@ -1,14 +1,13 @@
 """Apiclient for MSATP"""
 import json
-from stix_shifter_utils.stix_transmission.utils.RestApiClient import RestApiClient
+from stix_shifter_utils.stix_transmission.utils.RestApiClientAsync import RestApiClientAsync
 from datetime import datetime, timedelta
-import requests
 
 
 class APIClient:
     INCIDENTS_IDS_ENDPOINT = 'detects/queries/detects/v1'
     INCIDENTS_INFO_ENDPOINT = 'detects/entities/summaries/GET/v1'
-    TOKEN_ENDPOINT = 'https://api.crowdstrike.com/oauth2/token'
+    TOKEN_ENDPOINT = 'oauth2/token'
```

Comment:
Is `https://api.crowdstrike.com` not still needed?

Reply:
No, because previously the call was done via separate requests call (which is not asynchronous), and now for getting a token the same async self.client is used, which knows what is the host url

Reply:
though do you say that incident endpoints and token endpoint may have different host?

Reply:
I'm not sure on that to be honest. @mdazam1942 may know.

Reply:
confirmed working by running a ping with credentials we have
thanks @delliott90 and @mdazam1942 
marking this as resolved